src/components/header/HeaderLeft.tsx

```
@@ -12,7 +12,7 @@ export default function HeaderLeft() {
   const navigate = useNavigate();
   const isMobileSize = useMediaQuery({ query: "(max-width:430px)" });
 
-  const token = getToken();
+  const token = getUserToken();
   const getNameResponse = token ? useGetName() : null;
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)